tests/drivers/can/api/src/canfd.c

```diff
@@ -395,6 +496,24 @@ ZTEST_USER(canfd, test_set_timing_data_min)
 	zassert_equal(err, 0, "failed to start CAN controller (err %d)", err);
 }
 
+/**
+ * @brief Test setting a too low data phase bitrate.
+ */
+ZTEST_USER(canfd, test_set_bitrate_too_low)
+{
+	uint32_t min = can_get_bitrate_min(can_dev);
+	int err;
+
+	err = can_stop(can_dev);
+	zassert_equal(err, 0, "failed to stop CAN controller (err %d)", err);
+
+	err = can_set_bitrate_data(can_dev, min - 1);
```

Comment:
This will not do what you expect if minimum supported bitrate is zero.
Furthermore, this test is not CAN FD specific (it event uses classical bitrate settings) and does not belong in `canfd.c`.

Reply:
I add ztest_test_skip() when minimum supported bitrate is zero.